BE2_HuiswerkLes2/src/Main.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        int number;
+        String name;
+        int input1;
+        int input2;
+        hello();
+        positiveOrNegative(-1);
+        positiveOrZeroOrNegative(0);
+        bartender("Tom");
+        sum(4,20);
+
+        //BONUS-Opdrachten
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Voer een getal in: ");
+        int givenNumber = sc.nextInt();
+
+        Scanner sc1 = new Scanner(System.in);
+        System.out.println("Voer een text in: ");
+        String givenText = sc1.nextLine();
+
+        evenUnevenChecker(givenNumber);
+        textLengthChecker(givenText);
+    }
+
+    public static void hello() {
+        System.out.println("Hello World!");
+    }
+
+    public static boolean positiveOrNegative(int number) {
```

Comment:
Waarom return je hier een boolean? Je doet er uiteindelijk niks mee in main namelijk. In plaats van `public static boolean` kun je heir dan gewoon `public static void`  zetten

Reply:
Je had in deze methode een if/else kunnen maken (zonder else if). De waarde 0 had dan als positief of als negetiev gerekend, afhankelij kvan hoe je je if voorwaarde definieert.